02-analytics/technical.md

```diff
@@ -108,76 +118,52 @@ deliver additional data.
 
 | Data         | Name        | Comments                                       |
 |--------------|-------------|------------------------------------------------|
-| Name         | name        | A page identifier. See comments and list below |
-| Element Type | elementType | episode, series, ...                           |
-| Element ID   | elementId   | series, episode ID                             |
+| ID | Id | Code of the page if generated or from the list below. String |
+| Title | title | String, native language |
+| Additional info | meta | JSON. See below for `settings` example, but fields can be added as needed |
 
 ### Page/Screen IDs
 
-This are the currently used screens from the app:
-
-* AboutPage
-* AudiencePage
-* CalendarPage
-* ExplorePage
-* InfoPage
-* LiveStreamPage
-* LoginPage
-* NativePlayer
-* ProfileEditPage
-* ProfilePage
-* ProgramPage
-* QueuePage
-* SearchPage
-* SeriesPage
-* SettingsListPage
-* SupportPage
-
-## Category item click (category_item_click)
-
-*When*: User taps video on category page.
-
-*Reason*: 
-
-### API
-
-Use `/track` endpoint. Docs: https://docs.rudderstack.com/rudderstack-api/api-specification/rudderstack-spec/track
-
-### Data
-
-| Data             | Name            | Comments                                                                                   |
-|------------------|-----------------|--------------------------------------------------------------------------------------------|
-| Event ID         | event           | Hardcoded: `category_item_clicked`                                                         |
-| Category ID      | categoryId      | Category id of type int                                                                    |
-| Element Type     | elementType     | episode, series, ...                                                                       |
-| Element ID       | elementId       | id of the clicked element                                                                  |
-
-## Section rendered (video_slider_rendered)
-
-*When*: When items in horizontal collection that contains videos are rendered for the first time. This happens when a user visits the home page for the first time or scrolls down on said page. Featured corousel is not included (for now).
-
-*Reason*: This helps us see how what content the user has looked at, e.g. on the home page.
-
-### API
-
-Use `/track` endpoint. Docs: https://docs.rudderstack.com/rudderstack-api/api-specification/rudderstack-spec/track
+* about
+* calendar
+* livestream
+* login
+* profileEdit
+* profile
+* search
+* settings
+* support
+* faq
+* episode
```

Comment:
What do we use for the dynamic pages?

Reply:
The page code

Reply:
Is it important in terms of analytics? I just wanna know what page was visited, do I relly care about it being dynamic or not?

Reply:
yes, the generic pages like settings etc create noise when analyzing performance of those pages. Being able to filter them out by checking existence of a `pageCode` prop helps. 
"home" and "search" should include the extra "pageCode" to signal the dynamic page used.

Reply:
Ok, agree!

Reply:
added pageCode then